.github/workflows/main.yml

```diff
@@ -0,0 +1,61 @@
+# The name of the job, which will be displayed for the status check in PR.
+name: README and metadata formatting
+
+# Controls when the action will run.
+# Below triggers the workflow on pull requests to the specified branches.
+on:
+  pull_request:
+    branches:
+      - jen/test-status-checks
```

Comment:
This should be updated to `main` and we should add `dev`?

Reply:
Yes - helps if I push the change! Thanks